bindings/python/CMakeLists.txt

```diff
@@ -95,8 +95,7 @@ function(CREATE_PYTHON_TARGET target_name COMPILE_OPTIONS dependencies)
   endif(BUILD_WITH_OPENMP_SUPPORT)
 
   if(cereal_FOUND)
-    target_include_directories(${target_name} SYSTEM
-                               PRIVATE ${CEREAL_INCLUDE_DIRS})
+    target_link_libraries(${target_name} PRIVATE cereal)
```

Comment:
Yes, that change was necessary because `SYSTEM PRIVATE` is not possible for `target_link_libraries`.
But now, it is only a matter of modernizing CMake use (we avoid handling paths) and futureproofing (if cereal ever wants to add anything to its target, like definitions, we'll automatically have it).

Reply:
okay I understand, thanks for the clarification.